tests/sles4sap/ipaddr2/deploy.pm

```diff
@@ -39,7 +39,9 @@ sub run {
         diagnostic => get_var('IPADDR2_DIAGNOSTIC', 0),
         cloudinit => get_var('IPADDR2_CLOUDINIT', 1));
     $deployment{scc_code} = get_var('SCC_REGCODE_SLES4SAP') if (get_var('SCC_REGCODE_SLES4SAP'));
+    $deployment{trusted_launch} = 0 if (check_var('IPADDR2_TRUSTEDLAUNCH', 0));
```

Comment:
Just curious. Is additional bracket around `check_var` required here? 
``` $deployment{trusted_launch} = 0 if (check_var('IPADDR2_TRUSTEDLAUNCH', 0));```

Reply:
No are not. I like to always have `if ()` with braket.